apps/controllerx/cx_devices/aqara.py

```diff
@@ -572,6 +572,114 @@ def get_z2m_actions_mapping(self) -> DefaultActionsMapping:
         }
 
 
+class WXCJKG13LMMediaPlayerController(MediaPlayerController):
+    def get_z2m_actions_mapping(self) -> DefaultActionsMapping:
+        return {
+            "button_1_single": "click_volume_down",
+            "button_1_double": "mute",
+            # "button_1_triple": "", # Nothing
```

Comment:
Uncomment this, and assign it to `None`. You probably need to rebase the main branch in your current branch for this to work.

```python
"button_1_triple": None,
```

Reply:
Done.